src/components/RevokeContainer/RevokeDocumentDropZone/RevokeDocumentDropZone.tsx

```diff
@@ -30,14 +32,15 @@ export const RevokeDocumentDropZone: FunctionComponent<RevokeDocumentDropZone> =
 
   useEffect(() => {
     if (documentUploadState === DocumentUploadState.ERROR) {
-      const readFileError = new Error(
-        errorMessage ?? "Document cannot be read. Please check that you have a valid document"
-      );
-      setFileErrors([readFileError]);
+      const readFileError =
+        errorMessages !== undefined && errorMessages.length > 0
```

Comment:
how about setting `errorMessages` prop as required, so no need to check for undefined here?

Reply:
ok sure